.gitignore

```diff
@@ -101,4 +101,6 @@ dist
 .dynamodb/
 
 # TernJS port file
-.tern-port
\ No newline at end of file
+.tern-port
+
+build
```

Comment:
The contracts you deploy should be commited to the repo.

Reply:
listo